solr/core/src/test/org/apache/solr/response/transform/TestChildDocTransformerHierarchy.java

```diff
@@ -291,4 +302,15 @@ private static String generateDocHierarchy(int i) {
               "}\n" +
             "}";
   }
+
+  private static String IndexWoChildDocs() {
```

Comment:
First, never start a method name with an uppercase letter (at least in Java).  But secondly, I suggest inlining this to the point of use since you're only using it in one place.  I know this is a stylistic point but I find it harder to read code that refers to a bunch of other things in other places that I have to go read -- it's disjointed; disrupts reading flow.  Of course a good deal of that is normal (calling string.whatever) but here it's only used for this test.

Reply:
Oops how embarrassing, guess I was in a hurry :(.